dens-query/api/postgres/dens.sql

```diff
@@ -0,0 +1,116 @@
+-- References
```

Comment:
I see.

I'm going to hold on this until we get the changes from the changes discussed on slack in because that will shuffle the schema around a bit.